UNRELEASED.md

```diff
@@ -44,6 +44,7 @@ The autocomplete component is an input field that provides selectable suggestion
 
 ### Enhancements
 
+- Added `onScrollToBottom` prop to `Scrollable`
```

Comment:
Do we still add a link to the PR? @solonaarmstrong

Reply:
Yep, since it is all public now.

Reply:
The enhancements don't seem to have PR links.
Should I add it anyway?

<img width="864" alt="screen shot 2018-11-12 at 3 58 56 pm" src="https://user-images.githubusercontent.com/3925905/48374785-e3216e00-e693-11e8-9043-e3b70e2d27e5.png">

Reply:
Just rebased and all the enhancements have been released. So I added the PR link on mine.

Reply:
I removed the PR links in the changelog for v3.0.0 and earlier, because they would be linking to PRs in the old private codebase, which didn't seem particularly useful or accessible to people outside fo shopify.

Going forward we should keep PR links.

Reply:
Thank you for the clarification, I added the link!